src/core/dbc-gateway/queryMap.ts

```diff
@@ -10,11 +10,12 @@ export default {
   searchFacet: serviceUrlKeys.fbiSearch,
   searchWithPagination: serviceUrlKeys.fbiSearch,
   suggestionsFromQueryString: serviceUrlKeys.fbiSearch,
+  getMaterial: serviceUrlKeys.fbiSearch,
```

Comment:
Very nitpicky, but I think these `get...` lines were ordered alphabetically 🙈 But I see the rest isn't, so maybe who cares(?)

Reply:
No you're right. I made some changes to where getMaterial belonged and did not sort the lines afterwards. Iøll take care of it..